src/renderer/components/Transaction/TransactionTable.vue

```diff
@@ -246,9 +245,7 @@ export default {
 
   methods: {
     formatDate (value) {
-      moment.locale(window.navigator.userLanguage || window.navigator.language)
-
-      return moment(value).format('L HH:mm:ss')
+      return this.formatter_date(value)
```

Comment:
@alexbarnsley in case this should be in the 24h format like you said in the issue, we can pass it `L HH:mm:ss`

Reply:
I think its worth adding an option in the Profile Edit/Create screen to choose. Default it to locale I guess and people can change it that way